src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java

```diff
@@ -247,27 +273,31 @@ public void scan(HttpMessage msg, String param, String value) {
                     // This works out as a total of 2+4+4*1+4 = 14 reqs / param
                     nixCount = 2;
                     winCount = 4;
+                    dirCount = 4;
                     localTraversalLength = 1;
                     break;
 
                 case MEDIUM:
                     // This works out as a total of 4+8+4*3+4 = 28 reqs / param

```

Comment:
Really this was the original number of requests made by the plugin sinca a bit of year ago. I reviewed entirely this plugin last year to reduce these numbers and currently we test 1/3 of the original patterns!!!! The real trouble is that Path Traversal need a lot of checks and reducing too much them without following the criteria that have been implemented by Colm initially and by me next, reduce the possibility to find something according to the strenght because checks should be grouped unfortunately in similar families. I review the comments adding the new directory checks